pkg/cloud/cloud.go

```diff
@@ -149,7 +150,13 @@ func extractVersion(imageID string) (ver int, err error) {
 func GetImageID(options *flags.Options, version int) (name string) {
 	options.Release = removeDot(options.Release)
 	imageNamePrefix := imgTemplate(options)
-	return fmt.Sprintf("%s-%d-%s", imageNamePrefix, version, imageNameSufix)
+
+	finalVersion := strconv.Itoa(version)
+	if version == 0 {
```

Comment:
I would put a comment here that when version is 0, we are all snaps. For now, later this will likely change, so the comment will be handy.